src/lib/googleMaps.js

```diff
@@ -154,10 +156,33 @@ export async function createPolyline(shape) {
 	const polyline = new window.google.maps.Polyline({
 		path,
 		geodesic: true,
-		strokeColor: '#00FF00',
+		strokeColor: COLORS.POLYLINE,
 		strokeOpacity: 1.0,
 		strokeWeight: 4
 	});
 
 	return polyline;
 }
+
+/**
+ * Adds an arrow to the polyline.
+ * @param {google.maps.Polyline} polyline - The polyline to which the arrow will be added.
+ */
+export function addArrowToPolyline(polyline) {
+	const arrowSymbol = {
```

Comment:
Adding a check at the beginning of the function to validate the `polyline` parameter will be great, right @Ahmedhossamdev @aaronbrethorst? Something like this:

```
if (!(polyline instanceof google.maps.Polyline)) {
        console.error('Invalid polyline');
        return;
}
```

Reply:
Yep, that’s a great idea for checking the polyline parameter. Moving to TypeScript could also improve type safety and help as the project grows.